tests/utils.js

```diff
@@ -1,17 +1,40 @@
-/* global describe it expect */
-
+/* global describe before it expect */
 const expect = require('chai').expect;
```

Comment:
https://eslint.org/docs/rules/prefer-destructuring

Reply:
I'm removing this since it's not being used but you can add it back in as `const { expect } = require('chai');` if you need it in the future.